packages/server/migrations/1594758731889-migration.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
spaces between {} is missing

Reply:
Is lint really running for server? I don't think it should  pass